text_extensions_for_pandas/io/watson.py

```diff
@@ -0,0 +1,511 @@
+#
+#  Copyright (c) 2020 IBM Corp.
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+################################################################################
+# watson.py
+#
+# I/O functions related to Watson Natural Language Understanding on the IBM Cloud.
+# TODO: add links here and docstrings
```

Comment:
Can you please add the appropriate links here? Non-IBMers will not know where to look otherwise. Come to think of it, most IBMers wouldn't know either...